pkg/policies/opa/rego/aws/aws_vpc/vpcFlowLogsNotEnabled.rego

```diff
@@ -28,4 +28,16 @@ flowLogExist(vpc, vpc_input) = exists {
 	flow_log_vpcs_set := { vpc_id | input.aws_flow_log[i].type == "aws_flow_log"; vpc_id := split(input.aws_flow_log[i].config.vpc_id, ".")[1] }
 	flow_log_vpcs_set[vpc.name]
     exists = true
-}
\ No newline at end of file
+} else = exists {
+  	x:=vpc[i]
+    flow_log_vpcs_set := { vpc_id | vpc_mapping[x.id]; vpc_id := x.id}
+    flow_log_vpcs_set[x.id]
+    exists = true
+}
+
+vpc_mapping[vpc_id] = flow_vpc_id {
```

Comment:
this will break the design pattern, our policy engine can handle one return value from the rule. In your case, even if you look at the rego playground output, it is returning two outputs as if there are two rules in one single policy:

{
    "vpcFlowLogsNotEnabled": [],
    "vpc_mapping": {
        "vpc-0755f0db2302eeece": "vpc-0755f0db2302eeece",
        "vpc-0b346736ba3cfd21d": "vpc-0b346736ba3cfd21d"
    }
}
Here, only vpcFlowLogsNotEnabled is expected.

Reply:
Moreover, I realised else conditions are not making the policy readable. Along with this I tried to solve the issue addressed in this PR and solves for both cases: tfplan and normal output.
https://play.openpolicyagent.org/p/Zbs4CBUYRk.